### ISSUE TITLE
[Feature Request] Static Mod Time for Mount

### ISSUE THREAD

**Issue by drag0n141** [rclone/rclone#6986]
Hi,

i have found that FR https://github.com/rclone/rclone/issues/940.

I have now that problem again with Dropbox. It does not report the modtime of a folder. Problem is again that Plex always scans a movie when it thinks the folder has changed. Is it possible to build an option in with that we can set a static mount time?

**Comment by drag0n141** [comment]
Thats not correct, they said it does a refresh when the folder has changed.

https://forums.plex.tv/t/metadata-refreshing-when-i-select-a-movie/839771

**Comment by drag0n141** [comment]
Yes and every time this happens, Plex thinks there is a change.

**Comment by drag0n141** [comment]
@animosity22 thanks for Testing this again 😊 its not a show stopper thats correct. But an nice to have and reduce Plex refresh the Metadata.

**Comment by ncw** [comment]
Give this a go

[v1.63.0-beta.7003.7bf7d59ab.fix-6986-times](https://beta.rclone.org/branch/fix-6986-times/v1.63.0-beta.7003.7bf7d59ab.fix-6986-times/) on branch [fix-6986-times](https://github.com/rclone/rclone/tree/fix-6986-times) (uploaded in 15-30 mins)

### --default-time TIME ###

If a file or directory does have a modification time rclone can read
then rclone will display this fixed time instead.

The default is `2000-01-01 00:00:00 UTC`. This can be configured in
any of the ways shown in [the time or duration options](https://rclone.org/docs/#time-option).

For example `--default-time 2020-06-01` to set the default time to the
1st of June 2020 or `--default-time 0s` to set the default time to the
time rclone started up.

**Comment by drag0n141** [comment]
I have tested now the Branch, it does not work. Its always `"DefaultTime": {},` when i check the settings with `rclone rc options/get`
I have set it with `$(which rclone) rc options/set` and with `--default-time`

**Comment by animosity22** [comment]
Doesn't work for me either as the date just shows the mount time.

```
 rclone mount dcrypt-movies: /home/felix/test -vv --default-time 2023-01-01
```

```
rclone version
rclone v1.63.0-beta.7003.7bf7d59ab.fix-6986-times
- os/version: fedora 38 (64 bit)
- os/kernel: 6.2.14-300.fc38.x86_64 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.20.3
- go/linking: static
- go/tags: none
```

and

![image](https://user-images.githubusercontent.com/5088712/235904416-0337056f-a377-4246-9945-47a64be9b0b7.png)

**Comment by ncw** [comment]
Didn't change all the directory creations in the dropbox backend!

Try this

[v1.63.0-beta.7013.d05393b86.fix-6986-times](https://beta.rclone.org/branch/fix-6986-times/v1.63.0-beta.7013.d05393b86.fix-6986-times/) on branch [fix-6986-times](https://github.com/rclone/rclone/tree/fix-6986-times) (uploaded in 15-30 mins)

**Comment by drag0n141** [comment]
Looks good now its using your default setting `2000-01-01 00:00:00 UTC`
But Not the value i have set

**Comment by drag0n141** [comment]
I Set it over Rclone RC Options/Set, test again now 😊

**Comment by ncw** [comment]
> Looks good now its using your default setting `2000-01-01 00:00:00 UTC` But Not the value i have set

I see setting via the rc isn't working properly

Try this

[v1.63.0-beta.7014.67470ccff.fix-6986-times](https://beta.rclone.org/branch/fix-6986-times/v1.63.0-beta.7014.67470ccff.fix-6986-times/) on branch [fix-6986-times](https://github.com/rclone/rclone/tree/fix-6986-times) (uploaded in 15-30 mins)

Setting like this should work and it will read more sensibly too

```
$ rclone rc   options/set --json '{"main": {"DefaultTime": "2010-01-01"}}'
$ rclone rc   options/set --json '{"main": {"DefaultTime": "2000-02-01T00:00:00Z"}}'
$ rclone rc options/get | grep DefaultTime
		"DefaultTime": "2000-02-01T00:00:00Z",
```

**Comment by ncw** [comment]
Thanks for testing. I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.63
